lib/archive.ts

```diff
@@ -121,10 +121,10 @@ async function copySourceToDest(
   return false;
 }
 
-function cleanupTempDir(tmpDir: string): void {
+async function cleanupTempDir(tmpDir: string): Promise<void> {
   if (!tmpDir) return;
   try {
-    fs.remove(tmpDir);
+    await fs.remove(tmpDir);
```

Comment:
Same with this method.  [Docs](https://github.com/jprichardson/node-fs-extra/blob/master/docs/remove.md)